Source/DafnyLanguageServer/DafnyLanguageServer.csproj

```diff
@@ -16,7 +16,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Boogie.ExecutionEngine" Version="2.9.2" />
+    <PackageReference Include="Boogie.ExecutionEngine" Version="2.9.3" />
```

Comment:
For next time (don't want to block on it), we should be able to remove this line since it's redundant, correct?

Reply:
I don't actually know. I haven't changed this line in the past, but I needed to change it to build the DafnyLanguageServer. In which way is it redundant?

Reply:
Every project is already picking up the shared dependency declared in `Source/Directory.Build.props`.

I suspect the `DafnyLanguageServer` project was the first to depend on Boogie as a nuget dependency instead of from source, and then when the shared properties file was added we just didn't remove the now-redundant project-specific reference.

There's an outside change something in the packaging/release process is depending on it, so I didn't want to slow down this PR by being 100% sure about it first.